### PR TITLE
custom scroll bar

### PR DESCRIPTION
**Pull request by GeneralChauhan**
My submission for HacktoberFest

### CODE REVIEW

**Comment by GeneralChauhan** [comment]
I can add a bit more if you guys want!

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
 ### Simple devtips blog made with react and markup files [https://devtips-c17dd.web.app](https://devtips-c17dd.web.app)
+##more info about it
```
Comment (elukuro) [inline README.md]: I think we have to remove this additional information since this issue is related only to custom scroll bar

**Comment by elukuro** [comment]
@GeneralChauhan could you revert your change at README.md so I can approve your contribution for custom scroll bar

**Comment by GeneralChauhan** [comment]
Ok will do so!

**Review by GeneralChauhan — COMMENTED** [review]
I am deleting this

**Review by GeneralChauhan — COMMENTED** [review]
I am deleting this

**Comment by GeneralChauhan** [comment]
@elukuro  done bro!

**Pull request merged**